src/main/java/net/glowstone/GlowWorld.java

```diff
@@ -1809,13 +1826,25 @@ private void pulseTickMap() {
     public void requestPulse(GlowBlock block, long tickRate) {
         Location target = block.getLocation();
 
-
         if (tickRate > 0)
             tickMap.put(target, tickRate);
         else if (tickMap.containsKey(target))
             tickMap.remove(target);
     }
 
+    /**
+     * Calling this method will request that the block is ticked after the specified
+     * tix delay
+     *
+     * @param block The block to tick.
+     * @param tickRate The tick delay after which to pulse the block.
+     */
+    public void requestSinglePulse(GlowBlock block, long tickDelay) {
+        Location target = block.getLocation();
+
+        singleTickMap.put(target, worldAge + tickDelay);
```

Comment:
I don't see why we shouldn't. I honestly think negative tick doing delays would be a bad idea to do but i'm not sure there would be harm. Should I throw an exception if it's negative or just treat is as 1? It is sort of already being treated as 1 because it will just tick the next available time effectively being 1.

(If you think I shouldn't allow negative tickDelays should I allow 0 tick delays?)